packages/ui-components/src/components/relative-time-picker/relative-time-picker.config.ts

```diff
@@ -0,0 +1,434 @@
+			}
+		},
+		{
+			label: 'Last 30 days',
+			value: 'last-30-d',
+			comparisonConfig: ERelativeTimeComparisonConfig.RelativeAmountOfUnits,
+			startDate: {
+				amount: -30,
+				unit: 'days'
+			},
+			labelRangeFormatter: {
+				startDateFormatter: 'D MMM',
+				separator: 'to',
+				endDateFormatter: 'D MMM'
+			}
+		}
+	],
+	[
+		{
+			label: 'Last quarter',
+			value: 'last-1-q',
+			comparisonConfig: ERelativeTimeComparisonConfig.AbsoluteAmountOfUnits,
+			startDate: {
+				amount: -1,
+				unit: 'quarter'
+			},
+			endDate: {
+				amount: -1,
+				unit: 'quarter'
+			},
+			labelRangeFormatter: {
+				startDateFormatter: 'D MMM',
+				separator: 'to',
+				endDateFormatter: 'D MMM YYYY'
+			}
+		},
+		{
+			label: 'Last 90 days',
+			value: 'last-90-d',
+			comparisonConfig: ERelativeTimeComparisonConfig.RelativeAmountOfUnits,
+			startDate: {
+				amount: -90,
+				unit: 'days'
+			},
+			labelRangeFormatter: {
+				startDateFormatter: 'D MMM YYYY',
+				separator: 'to',
+				endDateFormatter: 'D MMM YYYY'
+			}
+		}
+	],
+	[
+		{
+			label: 'Last year',
+			value: 'last-1-y',
+			comparisonConfig: ERelativeTimeComparisonConfig.AbsoluteAmountOfUnits,
+			startDate: {
+				amount: -1,
+				unit: 'year'
+			},
+			endDate: {
+				amount: -1,
+				unit: 'year'
+			},
+			labelRangeFormatter: {
+				startDateFormatter: 'D MMM',
+				separator: 'to',
+				endDateFormatter: 'D MMM YYYY'
+			}
+		},
+		{
+			label: 'Last 365 days',
+			value: 'last-365-d',
+			comparisonConfig: ERelativeTimeComparisonConfig.RelativeAmountOfUnits,
+			startDate: {
+				amount: -365,
+				unit: 'days'
+			},
+			labelRangeFormatter: {
+				startDateFormatter: 'D MMM YYYY',
+				separator: 'to',
+				endDateFormatter: 'D MMM YYYY'
+			}
+		}
+	],
+	[
+		{
+			label: 'Last 2 years',
+			value: 'last-2-y',
+			comparisonConfig: ERelativeTimeComparisonConfig.AbsoluteAmountOfUnits,
+			startDate: {
+				amount: -2,
+				unit: 'year'
+			},
+			endDate: {
+				amount: -1,
+				unit: 'year'
+			},
+			labelRangeFormatter: {
+				startDateFormatter: 'D MMM YYYY',
+				separator: 'to',
+				endDateFormatter: 'D MMM YYYY'
+			}
+		},
+		{
+			label: 'Last 731 days',
+			value: 'last-731-d',
+			comparisonConfig: ERelativeTimeComparisonConfig.RelativeAmountOfUnits,
+			startDate: {
+				amount: -731,
+				unit: 'days'
+			},
+			labelRangeFormatter: {
+				startDateFormatter: 'D MMM YYYY',
+				separator: 'to',
+				endDateFormatter: 'D MMM YYYY'
+			}
+		}
+	],
+	[
+		{
+			label: 'All time',
+			value: 'all-time',
+			comparisonConfig: ERelativeTimeComparisonConfig.StartDate,
+			startDate: {
+				date: '1-01-2018'
+			},
+			labelRangeFormatter: {
+				startDateFormatter: 'D MMM YYYY',
+				separator: 'to',
+				endDateFormatter: 'D MMM YYYY'
+			}
+		}
+	]
+];
+
+export const CUSTOMIZE_OPTION_HEIGHT = 41;
+export const TIMEZONE_DROPDOWN_HEIGHT = 41;
+export const SELECT_OPTION_HEIGHT = 32;
+export const GROUP_GAP = 12;
+export const MAX_HEIGHT = 347;
```

Comment:
All these CSS options should be passed to the stylesheet in order to be synchronized. This will avoid weird behaviours if they ever get changed and we forgot to update them.